specification/integration_context.md

```diff
@@ -0,0 +1,86 @@
+<!-- markdownlint-configure-file 
+{ 
+  "MD013": { 
+    "tables": false 
+  } 
+}
+-->
+
+# Integration Context
+
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+In order to create a more complete picture of a distributed environment, we
+propose the use of additional headers to pass extra context details between
+Cisco observability components. Specifically, we are concerned with passing
+information between OpenTelemetry-based Splunk distributions and AppDynamics
+agents, in both directions.
+
```

Comment:
Providing product linkage in both directions, not passing data in both directions on the http request/response.  (Headers below are all request headers).

Reply:
Are you suggesting that I add some additional details about the purpose of passing data then here? I'm not sure I understand your comment.

Reply:
Just trying to clarify that the phrase "passing information... in both directions" might suggest that information is passed on http request and response, when it will not be.  Not a big deal.

Reply:
Thanks for clarifying. I tweaked it and remove the mention of "both directions". 👍🏻